chpl/chpl-api/e2e/collections/developer-controller.postman_collection.json

```diff
@@ -49,13 +98,202 @@
 			},
 			"response": []
 		},
+		{
+			"name": "developer schema and response validation_GET_/developers",
```

Comment:
Name of the request should only be request method and rest end points. 
Description mentioned above is covered in test name.

It applies to others as well.

Reply:
Leaving as is. There will be discussion on standard to write request name.